torch/testing/_internal/common_distributed.py

```diff
@@ -877,6 +888,66 @@ def _check_return_codes(self, elapsed_time) -> None:
     def is_master(self) -> bool:
         return self.rank == 0
 
+# Utility base class for DDP based Multi Process Test cases
```

Comment:
Why DDP based?

Reply:
You are right, it is meant to be distributed. Fixed in commit: 65e16deb07c5ca9acef323058a99d26e89633acb